packages/mutation-testing-report-schema/src/mutation-testing-report-schema.json

```diff
@@ -83,24 +125,140 @@
                   "description": "Actual mutation that has been applied.",
                   "examples": ["-", "+", "&&", "||"]
                 },
-                "description": {
-                  "type": "string",
-                  "description": "Description of the applied mutation.",
-                  "examples": ["removed call to java/io/Writer::write"]
-                },
-                "location": {
-                  "$ref": "#/definitions/location"
-                },
                 "status": {
                   "type": "string",
+                  "title": "MutantStatus",
                   "description": "Result of the mutation.",
                   "enum": ["Killed", "Survived", "NoCoverage", "CompileError", "RuntimeError", "Timeout", "Ignored"]
+                },
+                "testCoverage": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  }
+                },
+                "testsRan": {
+                  "type": "number",
+                  "description": "The number of tests actually ran for this mutant. Can differ from testCoverage because of \"--bail\" mode in test runners"
+                },
+                "duration": {
+                  "type": "number",
+                  "description": "The time it took to test this mutant in milliseconds."
                 }
               }
             }
           }
         }
       }
+    },
+    "performance": {
+      "type": "object",
+      "title": "PerformanceStatistics",
+      "description": "The performance statistics per phase. Total time should be roughly equal to the sum of all these.",
+      "properties": {
+        "setup": {
+          "type": "number",
+          "description": "Time it took to run the setup phase in milliseconds."
+        },
+        "initialRun": {
+          "type": "number",
+          "description": "Time it took to run the initial test phase (dry-run) in milliseconds."
+        },
+        "mutation": {
+          "type": "number",
+          "description": "Time it took to run the mutation test phase in milliseconds."
+        }
+      }
+    },
+    "system": {
+      "type": "object",
+      "title": "SystemInformation",
+      "description": "Information about the system that performed mutation testing",
+      "properties": {
+        "ci": {
```

Comment:
We should be careful with this value imo. The only possible values would be true or maybe-not. We can't definitively say that something did not run in a ci pipeline as we cannot support detection for all ci pipeline tools.

Reply:
And "yes" is also unknown, since I can run with `CI=TRUE` in my environment variables. I personally don't mind myself, I think it stands to reason that this value is "best-effort". I can clarify that in the description if you want.

Reply:
I think it is a good idea to clarify that in the description so anyone using the schema can share the same definitions.